build.clj

```diff
@@ -118,8 +118,8 @@
     (.mkdirs (io/file target-dir))
     (b/process {:command-args (concat ["clang" "-fpic" "-shared"]
                                       c-files
-                                      ["-o" test-c-library])}))
-  opts)
+                                      ["-o" test-c-library])})
+    opts))
```

Comment:
This is something I'd like reverted, the intention here is that `opts` is actually meant to be able to go to other build tasks, as run by `run-tasks`, so every task impl is supposed to return the unmodified opts at the end.

Reply:
oh, yeah i totally misunderstood the rationale here and reverted the changes already, but i borked those two lines up in the process.